### ISSUE TITLE
About reported performance on MPI-INF-3DHP in paper

### ISSUE THREAD

**Issue by dqj5182** [syguan96/BOA#7]
In the Table 4 of BOA paper, it is reported that BOA achieves PA-MPJPE of 77.4. However, when I run the MPI-INF-3DHP code from this repository, I get PA-MPJPE of 92.61. Is there reason for such difference in performance between main paper and this code?

![Screen Shot 2023-02-23 at 8 24 58 PM](https://user-images.githubusercontent.com/56655511/220892976-43f36e78-77e4-443f-9ff9-e97b930d5d1a.png)

**Comment by syguan96** [comment]
You should carefully tune loss weights and learning rate of BOA on 3DHP. During experiment, we also found that in-the-wild videos and 3DPW shares similar hyper parameters, but 3DHP not. The reason maybe that the labeled 3D keypoints and 2D keypoints are not so accurate, and the posture of 3DHP is much more hard than 3DPW. Hope this helps.

**Comment by dqj5182** [comment]
Is it possible I can get loss weights and learning rates of BOA on 3DHP (reported as in main paper)? It seems that there are too many hyperparameters for me to change in the system.

Thanks.

**Comment by syguan96** [comment]
I accidentally deleted it… sorry… One experiment only takes few minute. You can use grid search as I did. I remembered that betas of Adam also was tuned.From my phone在 2023年2月23日，20:54，Daniel Jung ***@***.***> 写道：﻿
Is it possible I can get loss weights and learning rates of BOA on 3DHP (reported as in main paper)? It seems that there are too many hyperparameters for me to change in the system.
Thanks.

**Comment by dqj5182** [comment]
Oh I see. No problem! I'll do the search on my own
